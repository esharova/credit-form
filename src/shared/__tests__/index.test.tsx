import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { App } from '../index';

test('Renders without crashing', () => {
  const component = renderer.create(
    <App />,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
