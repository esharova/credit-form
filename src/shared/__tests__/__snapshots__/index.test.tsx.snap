// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders without crashing 1`] = `
<div
  className="src_shared_index_css__test"
>
  Hello World! ðŸš€
</div>
`;
